cmd/notary/tuf.go

```diff
@@ -97,14 +97,22 @@ type tufCommander struct {
 	input  string
 	output string
 	quiet  bool
+
+	changes           []int
+	resetStatus       bool
```

Comment:
Since this pr has not been merged yet, would you mind if changing the variable name `changes` to `deletedIndex` and the `resetStatus` to `reset`?